### PR TITLE
Update icons in `MarkdownEditor`; add Help Link

### PR DESCRIPTION
**Pull request by lyzadanger**
This PR converts the icons in the `MarkdownEditor` component from CSS/icon font -> SVG. It also adds back a help link that goes to a knowledge-base article about formatting markdown.

After experimenting for a while with using the subcomponent `ToolbarButton` for the help link, I ended up using a link (`<a>` element) instead. CSS was a little bit wiggly—to get the right amount of space between the help icon and "Write/Preview" button, I had to do a bit of fiddly adjustment to margins and padding. I hope that's OK.

Before:

image

After:

image

Fixes https://github.com/hypothesis/client/issues/1452

### CODE REVIEW

**Comment by lyzadanger** [comment]
Hi, @robertknight ! Thanks for your attention to detail here. I completely overlooked the `@media` rules, derp.

This is round 2. I found it challenging to get the layout comfortable regarding the existing "Preview/Write" button and the help icon. You'll note that in the desktop view at this time, it looks a little "spacey":

image

But this is because of the click-/tap-target size that the "Preview" button takes up (as it should). This is easier to see with the "Preview" button focused:

Screen Shot 2019-11-21 at 10 16 21 AM

And the help button focused:

Screen Shot 2019-11-21 at 10 18 56 AM

When you look at it that way, they "take up" about the right amount of room. This uneven spac-y-ness can be "fixed" by grouping the help icon with the other formatting icons, but I feel like the help icon made sense where it was before, on the right/separate from formatting tools.

I do think the mobile variant is looking better now:

image

and, scrolled:

image

Thoughts?

**Comment by lyzadanger** [comment]
Oh, addendum: Part of the struggle here is that the dimensions of the help-icon SVG are proportionally different from the other SVG icons. In this round, I worked around that by having a `min-height` and `min-width` on the desktop-variant buttons as well as the mobile ones. Shrug.

File: src/sidebar/components/markdown-editor.js [hunk 1]
```diff
@@ -139,60 +145,73 @@ function Toolbar({ isPreviewing, onCommand, onTogglePreview }) {
...
         shortcutKey={SHORTCUT_KEYS.list}
         title="Bulleted list"
       />
       <span className="u-stretch" />
+      <div className="markdown-editor__toolbar-help-link">
+        <a
+          href="https://web.hypothes.is/help/formatting-annotations-with-markdown/"
+          target="_blank"
+          rel="noopener noreferrer"
+          className="markdown-editor__toolbar-button"
```
Comment (robertknight) [inline src/sidebar/components/markdown-editor.js]: This link could use a title/accessible label.

Comment (lyzadanger) [reply src/sidebar/components/markdown-editor.js]: Great catch—done!

**Pull request merged**